frontend/src/components/where-to-buy/where-to-buy.tsx

```diff
@@ -0,0 +1,74 @@
+import { useMemo, useState } from 'react';
+
+import { ButtonOutline } from '@components/common/button-outline/button-outline';
+import { useAppSelector } from '@hooks/store/store.hooks';
+import SearchIcon from '@mui/icons-material/Search';
+import { useGetWhereBuyQuery } from '@store/queries/where-buy';
+import { clsx } from 'clsx';
+
+import styles from './styles.module.scss';
+import { WhereBuyItem } from './where-buy-item/where-buy-item';
+
+interface WhereToBuyProps {
+  complectationId: string;
+}
+
+const WhereToBuy: React.FC<WhereToBuyProps> = ({ complectationId }) => {
+  const [page, setPage] = useState(0);
+  const [isSorted, setIsSorted] = useState(false);
+  useGetWhereBuyQuery({
+    page,
+    complectationId,
+    countpage: '20',
```

Comment:
Use variable here. Maybe in future we would provide section how many items on the page user want to see